### ISSUE TITLE
trim Function to remove leading and trailing whitespace?

### ISSUE THREAD

**Issue by KonradHoeffner** [tarql/tarql#53]
Does tarql 1.1 offer a trim function to remove leading and trailing whitespace? If not I request it as a new feature.

**Comment by cygri** [comment]
No, there is no trim function, but I suppose this can be done fairly easily with the standard SPARQL `REPLACE` function, which accepts a regular expression.

Ideally, Tarql would have a way of defining custom functions based on SPARQL expressions, similar to SPIN functions. This would make it possible to package the `REPLACE` call into a simple `my:trim(?x)` call.

**Comment by VladimirAlexiev** [comment]
#34 asks for custom SPARQL functions
